src/app/main.cpp

```diff
@@ -187,6 +189,19 @@ int main(int argc, char *argv[])
         std::cerr << "Couldn't set environment variable...\n";
 
 #ifndef DISABLE_GUI
+
+    // Set dark theme
+    if (params.darkTheme) {
+        QFile file(":/qdarkstylesheet.qss");
+        if(file.open(QIODevice::ReadOnly | QIODevice::Text)) {
```

Comment:
There is a config for uncrustify which can help you at least partially. The config is not complete yet, and you are welcomed to extend it, of course.